frontend/ui/widget/confirmbox.lua

```diff
@@ -41,13 +41,15 @@ local Screen = Device.screen
 
 local ConfirmBox = InputContainer:new{
     modal = true,
+    keep_dialog_open = false,
     text = _("no text"),
     face = Font:getFace("infofont"),
     ok_text = _("OK"),
     cancel_text = _("Cancel"),
     ok_callback = function() end,
     cancel_callback = function() end,
     other_buttons = nil,
+    other_buttons_above = false, -- set to true to place other buttons above Cancel-OK row
```

Comment:
(nitpick about the wording, not even sure about my feeling: "_above" expects a "what" - I would have used "_first" that doesn't need a what :)